asv.conf.json

```diff
@@ -1,14 +1,14 @@
 {
-    // See https://asv.readthedocs.io/en/stable/asv.conf.json.html for documentation.
```

Comment:
Why drop this link? Is ASV sufficiently well-known that readers can be expected to know what this file is without it?